src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/arraytrailingcomma/InputArrayTrailingComma.java

```diff
@@ -38,4 +38,7 @@
 
     int[] e1 = new int[] {
     };
+
+    int[] f1 = new int[] {0, 1
+    };
```

Comment:
please add test case with comma in inner array (the same as in issue description):
```
     int[][] d2 = new int[][] 
     {
         {1, 
           2, }, 
     }
```